js/src/controls/WidgetControl.js

```diff
@@ -0,0 +1,94 @@
+var widgets = require('@jupyter-widgets/base');
+var _ = require('underscore');
+var L = require('../leaflet.js');
+var control = require('./Control.js');
+var LeafletControlView = control.LeafletControlView;
+var LeafletControlModel = control.LeafletControlModel;
+
+L.Control.WidgetControl = L.Control.extend({
+    options: {
+        position: 'topleft',
+        widget : 'null',
+    },
+
+    _updateContent: function () {
+        if (!this._content) { return; }
+        var node = this._container
+        L.DomEvent.disableClickPropagation(node);
+        L.DomEvent.disableScrollPropagation(node);
```

Comment:
Maybe you can move those two lines:
```javascript
L.DomEvent.disableClickPropagation(node);
L.DomEvent.disableScrollPropagation(node);
```
in the `onAdd` method? After `this._container` is created? So that you call it only once.